proxy/src/bind.rs

```diff
@@ -60,7 +63,13 @@ pub enum Host {
     NoAuthority,
 }
 
-pub type Service<B> = Reconnect<NewHttp<B>>;
+/// Reconstruct HTTP/1.x URIs.
```

Comment:
What does this mean? Are requests modified? If so how? If not, what does the middleware actually do?